acnportal/acnsim/events/event.py

```diff
@@ -76,19 +88,29 @@ def _from_dict(
         return out_obj, loaded_dict
 
 
-class PluginEvent(Event):
-    """ Subclass of Event for EV plugins.
+class EVEvent(Event):
+    """ Subclass of Event for events which deal with an EV such as Plugin and Unplug
+    events.
 
     Args:
         timestamp (int): See Event.
-        ev (EV): The EV which will be plugged in.
+        ev (EV): The EV associated with this event.
     """
+    ev: "EV"
 
-    def __init__(self, timestamp, ev):
+    def __init__(self, timestamp: int, ev: "EV") -> None:
         super().__init__(timestamp)
-        self.event_type = "Plugin"
         self.ev = ev
-        self.precedence = 10
+
+    @property
+    def station_id(self) -> str:
+        """ Return the station_id for the EV associated with this Event. """
+        return self.ev.station_id
+
+    @property
+    def session_id(self) -> str:
+        """ Return the session_id for the EV associated with this Event. """
+        return self.ev.session_id
```

Comment:
@zach401 session/station_id accessors are in EVEvent, is this okay or should I put them in UnplugEvent?

Reply:
I think they are fine in EVEvent. They don't hurt anything for PluginEvents.